### PR TITLE
Web console: Add frontend buttons to remove group by

### PR DESCRIPTION
**Pull request by renevan10**
This is a follow-up to the toolkit PR: [Remove Group By](https://github.com/implydata/druid-query-toolkit/pull/23)

Provides buttons for removing a group-by column.

Screen Shot 2019-09-13 at 3 37 03 PM

### CODE REVIEW

File: web-console/src/views/query-view/column-tree/column-tree-menu/string-menu-items/string-menu-items.spec.tsx [hunk 1]
```diff
@@ -37,4 +37,17 @@ describe('string menu', () => {
     const { container } = render(stringMenu);
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('matches snapshot containing remove group by button', () => {
+    const stringMenu = (
+      <StringMenuItems
+        columnName={'channel'}
+        parsedQuery={parser(`SELECT channel, count(*) as cnt FROM wikipedia GROUP BY 1`)}
```
Comment (vogievetsky) [inline web-console/src/views/query-view/column-tree/column-tree-menu/string-menu-items/string-menu-items.spec.tsx]: How is this test case different from the one above? I think you do not need to add new snapshot tests (they do not add anything as far as I can see). You need to update the existing snapshots with `jest -u`

Comment (renevan10) [reply web-console/src/views/query-view/column-tree/column-tree-menu/string-menu-items/string-menu-items.spec.tsx]: I've made changes to the tests per our discussion.

**Comment by vogievetsky** [comment]
image

Could you put the remove group by function after the group by?

**Comment by vogievetsky** [comment]
LGTM after CI

**Comment by vogievetsky** [comment]
Thank you for the contribution!

**Pull request merged**